examples/pytorch_lightning_distributed/simmim.py

```diff
@@ -75,21 +73,16 @@ def configure_optimizers(self):
 device = "cuda" if torch.cuda.is_available() else "cpu"
 model.to(device)
 
-# we ignore object detection annotations by setting target_transform to return 0
-pascal_voc = torchvision.datasets.VOCDetection(
-    "/home/ubuntu/datasets/pascal_voc", download=False, target_transform=lambda t: 0
-)
 transform = MAETransform()
-dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=transform)
+dataset = pascal_voc = torchvision.datasets.VOCDetection(
+    "datasets/pascal_voc", download=True, transform=transform
+)
```

Comment:
```suggestion
dataset = torchvision.datasets.VOCDetection(
    "datasets/pascal_voc", download=True, transform=transform
)
```

Is `target_transform` not needed anymore?